### PR TITLE
Convert entity command help summaries to use third-person singular verbs.

### PR DESCRIPTION
**Pull request by DrewAPicture**
Per wp-cli/wp-cli#4542, this PR converts help docs for entity commands and subcommands, respectively, to use third-person singular verbs.

### CODE REVIEW

File: src/User_Command.php [hunk 1]
```diff
@@ -1030,7 +1030,7 @@ private static function validate_role( $role ) {
 	}
 
 	/**
-	 * Acommodate three different behaviors for wp_new_user_notification()
+	 * Acommodates three different behaviors for wp_new_user_notification()
```
Comment (schlessera) [inline src/User_Command.php]: There was a typo in the original description.
This should be `Accommodates` (2 `c`s).

Comment (DrewAPicture) [reply src/User_Command.php]: Fair enough ;)

I've also updated the PR from latest `master`.

**Comment by danielbachhuber** [comment]
@schlessera README needs to be regenerated before merge.

**Comment by danielbachhuber** [comment]
> README needs to be regenerated before merge.

I've updated

**Comment by DrewAPicture** [comment]
I tried to figure out how to do it but came up short.

**Comment by danielbachhuber** [comment]
@DrewAPicture No worries. Details are here: https://github.com/wp-cli/scaffold-package-command/#wp-scaffold-package-readme

**Pull request merged**